release-notes/v0.6.0.yaml

```diff
@@ -0,0 +1,75 @@
+date: Nov 1, 2023
+
+changes:
+  - area: documentation
+    change: |
+      Introduced a new website based on Hugo
+      Added Grafana dashboards and integration docs for EnvoyProxy metrics
+      Added Grafana integration docs for Gateway API metrics
+
+  - area: installation
+    change: |
+      Updated EnvoyProxy image to be a distroless variant.
+      Removed resources around kube-rbac-proxy
```

Comment:
miss https://github.com/envoyproxy/gateway/pull/2108?

Reply:
good call, I added it to the Release announcement but missed it in the notes

Reply:
looks like there is already a note in there about `kube-rbac-proxy`